source/_landing.erb

```diff
@@ -0,0 +1,31 @@
+<div class="info landing window">
+  <div id="top">
+    <a id="close" href="">&times;</a>
+  </div>
+
+  <greeting>你好 ni hao!</greeting>
+
+  <description>
+    Hanzi Maker is part of Counter Algorithmic Censorship Toolkit.<br><br>
+
+    Censorship in China is speading beyond its border. It is affecting everyone, including you. (insert sentences about why people will find this tool useful) We invite you to create your own Hanzi (Chinese charachter) to bypass automated censorship. <br><br>
+
+    If you have no prior knowledge about Hanzi, we recommend you to go through the Hanzi 101 session.<br><br>
+
+    Note: The data generated during your participation will not be stored. <br>
+    <br>
+    <br>
+    <br>
+
+    <br>
+  </description>
+
+  <div class='info-footer'>
+      <a href="http://www.qianqian-ye.com/Everyday/Day47/index.html"><button>Hanzi 101</button></a>
```

Comment:
Right now this link makes onboarding the whole page. I'm thinking through one way you could show the onboarding partial. You could load the onboarding partial and the landing page at the same time, but have the onboarding divs hidden. Then you can have the button "Hanzi 101" show all the onboarding divs. Does that make sense?